tests/e2e/cucumber/features/integrations/reshare.ocis.feature

```diff
@@ -0,0 +1,46 @@
+Feature: reshare
+
+  Background:
+    And "Admin" disables share auto accepting
+
```

Comment:
I noticed the name of the file is `reshare.ocis.feature` if it's only going to be run for ocis I don't think we need this line? Is auto accept configurable in OCIS? Maybe we can run this test for both backend. `reshare.feature` ?
```suggestion
    Given "Admin" disables share auto accepting

```

Reply:
Thanks, I implemented api for both backends, but forgot to change file name